wfe/wfe.go

```diff
@@ -49,12 +49,10 @@ const (
 	newRegPath    = "/acme/new-reg"
 	regPath       = "/acme/reg/"
 	newAuthzPath  = "/acme/new-authz"
-	authzPath     = "/acme/authz/"
```

Comment:
:tada: :rocket: 

Removing the handlers and related code from WFE / WFE2 also seems like a good chunk to split off into its own PR.